protocol/contracts/libraries/Silo/LibTokenSilo.sol

```diff
@@ -259,9 +259,8 @@ library LibTokenSilo {
             return removedBDV;
         }
         // Full remove
+        // deletes the deposit from storage
```

Comment:
combine 261 and 262 imo

Reply:
of course, fixed [here](https://github.com/BeanstalkFarms/Beanstalk/pull/736/commits/4a67fe7ca1d7cb2b4ac4da1922879d96ad079221)